test/e2e/kubernetes/networkpolicy/networkpolicy.go

```diff
@@ -60,14 +61,14 @@ func CreateNetworkPolicyFromFile(clientset *kubernetes.Clientset, filename, name
 }
 
 func createNetworkPolicyV1(clientset *kubernetes.Clientset, namespace string, networkPolicy *networkingv1.NetworkPolicy) error {
-	_, err := clientset.NetworkingV1().NetworkPolicies(namespace).Create(networkPolicy)
+	_, err := clientset.NetworkingV1().NetworkPolicies(namespace).Create(context.Background(), networkPolicy, metav1.CreateOptions{})
```

Comment:
Please pipe through a context, or create an issue to follow up on piping through contexts for each place a `context.Background()` was added. Each one is an opportunity for the func to block indefinitely (or until the clientset timeout occurs).

Reply:
yes will do that